projects/core/src/lib/operators/dialogs/class-histogram-operator/class-histogram-operator.component.ts

```diff
@@ -40,7 +40,8 @@ const categoricalInputValidator =
             return projectService.getRasterLayerMetadata(layer).pipe(
                 first(),
                 map((metadata) => {
-                    if (metadata.measurement instanceof ClassificationMeasurement) {
+                    const allBandsAreClassification = metadata.bands.every((band) => band instanceof ClassificationMeasurement);
```

Comment:
will we allow that a raster stack sonsists of classifications and other things?

Reply:
I don't know but I wouldn't rule mixed stacks out right now. I think there are lots of use cases that need this.